spec-main/api-web-contents-spec.ts

```diff
@@ -1541,6 +1541,39 @@ describe('webContents module', () => {
     });
   });
 
+  describe('getBackgroundThrottling()', () => {
```

Comment:
nit: you test are methods and setters and getter here, so can name this suite "Background throttling".